backend-rust/src/bin/ccdscan-api.rs

```diff
@@ -0,0 +1,93 @@
+use anyhow::Context;
+use clap::Parser;
+use concordium_scan::graphql_api;
+use dotenv::dotenv;
+use sqlx::PgPool;
+use std::{
+    net::SocketAddr,
+    path::PathBuf,
+};
+use tokio::net::TcpListener;
+use tokio_util::sync::CancellationToken;
+use tracing::{
+    error,
+    info,
+};
+
+// TODO add env for remaining args.
+#[derive(Parser)]
+struct Cli {
+    /// The URL used for the database, something of the form
+    /// "postgres://postgres:example@localhost/ccd-scan"
+    #[arg(long, env = "DATABASE_URL")]
+    database_url: String,
```

Comment:
Keep in mind that command-line arguments are not private (can be seen by other processes) so only using an env var for this might be preferable as this URL could contain a password.

Reply:
Good point, it is convenient through, so I'll add a warning in the documentation for now.